src/components/SearchableSingleSelect/SearchableSingleSelect.spec.jsx

```diff
@@ -57,6 +57,9 @@ describe('SearchableSingleSelect', () => {
 					props: {
 						callbackId: 'zero',
 						children: 'Zero',
+						isDisabled: false,
```

Comment:
Aren't these three props already defaulted? Did they need to be added here?

Reply:
IIRC I was getting ts errors if I didn't supply these props, they were coming through on the received props